File: Server/Game/Models/PlayerModel.cs
using Server.Game.Entities;

namespace Server.Game.Models;

public class PlayerModel(Player p)
{
    public double PosX { get; set; } = p.PosX;
    public double PosY { get; set; } = p.PosY;
    public string Id { get; set; } = p.Id;
    public int Lives { get; set; } = p.LifeAmount();
}

File: Client/template-nextjs/src/game/Player/PlayerModel.ts
import { Player } from "./Player"

export class PlayerModel {
    
    id : string
    name : string
    posX : number
    posY : number
    x : number
    y : number
    dead : boolean
    skin : string
    sprite : Phaser.GameObjects.Sprite

    constructor(id : string, name : string, x : number, y : number, skin : string) {
        this.id = id;
        this.posX = x;
        this.posY = y;
        this.name = name;
        this.skin = skin;
      }

      public skinUp()
      {
        //this.skin++;
      }

      public skinDown()
      {
        //this.skin--;
      }
}